src/Heist/Internal/Types/HeistState.hs

```diff
@@ -237,20 +238,20 @@ instance Exception CompileException
 -- HeistState@ \"filter\" functions.  Then you use the resulting @HeistState@
 -- in calls to 'renderTemplate'.
 --
--- m is the runtime monad
-data HeistState m = HeistState {
+-- n and m are the runtime and load time monads, respectively
+data HeistState n m = HeistState {
```

Comment:
Snap has a strong commitment to backwards compatibility as much as possible, so it would be really nice if this could be done without breaking the type signature here and in `HeistConfig` and `SpliceConfig`.  Usually we like to have a deprecation cycle to give people more time to upgrade before we spring the breaking changes.  Not sure how feasible this would be though.